packages/components/src/components/banner/banner.tsx

```diff
@@ -67,24 +67,26 @@ export class Banner {
 
   public render(): JSX.Element {
     validateProps(this, propTypes);
-    attachComponentCss(this.host, getComponentCss);
+    const deprecatedStateMap: Partial<Record<InlineNotificationState, InlineNotificationState>> = {
+      neutral: 'info',
+    };
+    warnIfDeprecatedPropValueIsUsed(this.host, 'state', deprecatedStateMap);
+    attachComponentCss(this.host, getComponentCss, this.width);
```

Comment:
Naming of `width`  should probably be aligned with what's going on in the carousel since there is similar behavior.

Reply:
added it to v3 status docs